solr/docker-compose.yml

```diff
@@ -13,12 +12,4 @@ solr:
   volumes:
     - ./env/solr/core01:/opt/solr/server/solr/core01
     - ./env/solr/core02:/opt/solr/server/solr/core02
-  volumes_from:
-    - solr_data
-# This container exists to allow for setting the data area of solr to
-# a persistant storage area and fixing permissions so that the solr
-# container can write to it. This does not need to be done per core
-solr_data:
-  build: ./dockerfiles/solr_data
-  volumes:
-    - /data/solr_example/solr:/var/lib/solr
+    - /data/outrigger/solr:/var/lib/solr
```

Comment:
The permission probably comes from this, based on how this is the major variation between what you have and what I've recently been using. This can inherit some directory level permissions of some sort from the source side.

Instead, we could define this as a named volume.

```
solr-data:/var/lib/solr
```

Then parallel to the `services` key:

```
volumes:
  # This data volume persists Solr storage.
  # The search service is using a customized Docker image to facilitate write
  # access to this volume.
  # An empty value indicates all settings are default.
  solr-data:
```